### ISSUE TITLE
state.emitSelection is not a function error

### ISSUE THREAD

**Issue by Kliton** [alpertuna/react-metismenu#22]
Hello, i have this issue:

```
Uncaught TypeError: state.emitSelected is not a function
    at emitters (emitters.js:19)
    at combination (combineReducers.js:132)
    at combination (combineReducers.js:132)
    at dispatch (createStore.js:179)
    at Object.activateMe [as onClick] (Item.js:34)
    at Object.handleClick (Link.js:97)
    at Object.ReactErrorUtils.invokeGuardedCallback (ReactErrorUtils.js:70)
    at executeDispatch (EventPluginUtils.js:85)
    at Object.executeDispatchesInOrder (EventPluginUtils.js:108)
    at executeDispatchesAndRelease (EventPluginHub.js:43)
```

Here is my code
```javascript
import React from 'react';
import { render } from 'react-dom';
import { Router, Route, hashHistory } from 'react-router';
import MetisMenu from 'react-metismenu';
import RouterLink from 'react-metismenu-router-link';
import Flexbox from 'flexbox-react';

const Menu1 = () => <div><u>Menu 1 View</u></div>;
const Menu2 = () => <div><i>Menu 2 View</i></div>;
const SubMenu = () => <div><s>SubMenu View</s></div>;

const menu = [
  {
    icon: 'dashboard',
    label: 'Menu 1',
    to: 'menu-1',
  },
  {
    icon: 'bell',
    label: 'Menu 2',
    to: 'menu-2',
  },
  {
    icon: 'bolt',
    label: 'Menu 3',
    content: [
      {
        icon: 'bolt',
        label: 'Sub Menu',
        to: 'sub-menu',
      },
    ],
  },
  {
    icon: 'external-link',
    label: 'External Link',
    externalLink: true,
    to: 'https://www.google.com',
  },
];

const App = (props) => (
  <Flexbox flexDirection="row">
    <Flexbox element="aside" width="100px" height="100%">
      <MetisMenu
        activeLinkFromLocation
        content={menu}
        LinkComponent={RouterLink}
      />
      </Flexbox>
      <Flexbox flexGrow={1}>

    <h2>Page Content</h2>
    {props.children || "Greeter Page"}
  </Flexbox>
  </Flexbox>
);

render(
  <Router history={hashHistory}>
    <Route path="/" component={App}>
      <Route path="menu-1" component={Menu1} />
      <Route path="menu-2" component={Menu2} />
      <Route path="sub-menu" component={SubMenu} />
    </Route>
  </Router>,
  document.getElementById('app')
);
```

**Comment by rizkiandrianto** [comment]
up, I also have the same problem.

**Comment by alpertuna** [comment]
Hi,
Yes, there was a little bug. However, it is fixed with `v1.3.0-alpha.2`.
Thanks for your report.
